tensorflow-tools/src/main/java/org/tensorflow/tools/ndarray/NdArray.java

```diff
@@ -293,4 +291,20 @@ default long size() {
    * @see DataBuffer#size()
    */
   NdArray<T> write(DataBuffer<T> src);
+
+  /**
+   * Checks equality between n-dimensional arrays.
+   *
+   * <p>An array is equal to another object if this object is another {@link NdArray} of the
```

Comment:
This should probably note "the same shape, type and the elements are equal and in the same order".